### DISCUSSION TITLE
Issue with docker-compose install...

### DISCUSSION THREAD

**Discussion by nin9creative** [directus/directus#12676] [Archive]
### Describe the Bug

I was following the directions listed here: https://docs.directus.io/getting-started/installation/docker/ using the docker compose method.

Everything spins up and I can get to the login screen, however no matter what I set for admin user/password - I'm not able to login.

I'm wondering if there is some issue with the database connectivity or something else from the front end.

I would have figured that it would work out of the box following the example.

### To Reproduce

Install using docker compose example: https://docs.directus.io/getting-started/installation/docker/

Run

Try to login with set environment variable admin user / password

Error

### Errors Shown

![image](https://user-images.githubusercontent.com/449883/162519437-01b5f8ef-7726-4d70-99f3-9b763786ad4a.png)

<details>
<summary>More Info</summary>

### What version of Directus are you using?

9.8.0

### What version of Node.js are you using?

16.14.2

### What database are you using?

Postgres 13

### What browser are you using?

Chrome

### How are you deploying Directus?

Docker on Azure Ubuntu VM

</details>

**Comment by azrikahar** [comment]
Moved this to discussion as I believe this is a user config issue.

---

> I would have figured that it would work out of the box following the example.

It does! I've also tested the exact same docker-compose in a DO ubuntu droplet just to be extra sure, and it works as seen here:

https://user-images.githubusercontent.com/42867097/162715886-9ca911be-bd2d-4aac-ab27-cd696d7914e2.mp4

Note: I've since then removed the domain used in the clip so it will not work anymore in case anyone is curious :)

---

One possibility I can think of is there is a bind mount for the database:

```yaml
volumes:
      - ./data/database:/var/lib/postgresql/data
```

So you might be running into the issue of the database is still using the _first_ credentials, possibly randomized if you didn't set `ADMIN_EMAIL` and `ADMIN_PASSWORD` initially, so any subsequent attempts that you are doing without wiping that volume will only result in wrong credentials. But once again I do believe this is not something to do with Directus itself, more so a config issue or misalignment of expectations with the tools used. Do let us know if it turns out to not be the case!
